app/src/main/java/com/hercan/weatherapp2/presentation/model/CityWeatherItem.kt

```diff
@@ -0,0 +1,21 @@
+package com.hercan.weatherapp2.presentation.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class CityWeatherItem(
+    val city: String,
+    val temperature: Int,
+    val temperatureRange: String,
+    val weather: String
+) : Parcelable
+
+fun CityWeatherItem.getTemperatureRange(): String {
+    val temperatureRange = this.temperatureRange.split(" - ")
```

Comment:
first ve last icin anlamli degisken isimleri kullanilabilir. 
```suggestion
    val temperatureRange = this.temperatureRange.split(" - ")
    val temperatureMin = temperatureRange.first()
    val temperatureMax = temperatureRange.last()
```
ya da min ve max olarak ayri degiskenlerde tutulabilir.